tests/unit/auth/AuthInfo.js

```diff
@@ -72,4 +72,20 @@ describe('AuthInfo class constructor', () => {
         const publicUser = new AuthInfo({ canonicalID: constants.publicId });
         assert.strictEqual(publicUser.isRequesterPublicUser(), true);
     });
+
+    it('should have a working isRequesterAServiceAccount() method', () => {
+        assert.strictEqual(authInfo.isRequesterAServiceAccount(), false);
+        const serviceAccount = new AuthInfo({
+            canonicalID: `${constants.zenkoServiceAccountsNs}/Clueso` });
+        assert.strictEqual(serviceAccount.isRequesterAServiceAccount(), true);
+    });
+
+    it('should have a working isRequesterThisServiceAccount() method', () => {
+        const serviceAccount = new AuthInfo({
+            canonicalID: `${constants.zenkoServiceAccountsNs}/Clueso` });
```

Comment:
I think we should do lowercase since we're doing a sort of url

Reply:
I agree, will change the test.